### ISSUE TITLE
Moving Zim Files from Download folder to another Drive/Folder

### ISSUE THREAD

**Issue by marcelb3** [kiwix/kiwix-apple#859]
All of my Zim files, which were downloading from KIWIX, are stored in my Downloads folder.  I want to move them to a different drive and folder and then let KIWIX know to use the new location.   How do I do that?

**Comment by kelson42** [comment]
@marcelb3 What about moving the files manually and the reload them one by one?

**Comment by BPerlakiH** [comment]
If this is for macOS, you have 2 options: 
**Option 1)** 
a loaded file can be manually moved to a new place outside of Kiwix. The best is to close Kiwix for this operation. Once you moved the file, and re-open Kiwix, in the Library / Opened section you will see a yellow rectangle, indicating that the file cannot be found. Once you select it you will have the option to "Locate" it:

<img width="916" alt="Screenshot 2024-07-10 at 20 09 11" src="https://github.com/kiwix/kiwix-apple/assets/6784320/96767589-4686-4baa-bd0f-cdde8b660c0b">

If you point it to the new location, it should start working again as before.

**Option 2)**
Under Library/Opened you can select the ZIM file, and "Unlink" it. This will remove everything Kiwix "knows" about this file, (including where it can be found). It will not delete the file itself. You can then move the file to a new location, double click on the file and Kiwix will "re-import" it. For this option you don't even need to close Kiwix.
<img width="895" alt="Screenshot 2024-07-10 at 20 10 54" src="https://github.com/kiwix/kiwix-apple/assets/6784320/6f4d4e34-ff74-47ad-92e2-d7cfc053f385">

The real difference between Option 1 and Option 2: 
- If you have bookmarks associated with the file, go for Option 1, as this way they will be preserved.
- if it's a fresh ZIM file, without any bookmarks, Option 2 will be equally good.

**Comment by kelson42** [comment]
@marcelb3 All good? Can I close the issue?

**Comment by garfnodie** [comment]
Why the convoluted solution when you could just give us the option to set a custom download location?

**Comment by BPerlakiH** [comment]
@garfnodie there are some limitations on what Apple allow us to do and what not, when it comes to downloads.
There were discussion / attempts to solve this for only macOS by using a different kind of download solution here: https://github.com/kiwix/kiwix-apple/issues/1549

TLDR: we also wish it was easier to support such cases than that, but it's not.

**Comment by garfnodie** [comment]
I've installed other stuff from github, homebrew, etc, and they all seem to be able to do it.  What's the limitation that you're hitting that other OSS programs seem to able to handle?

**Comment by BPerlakiH** [comment]
homebrew is not something you install from the App Store, therefore the same "rules" won't apply.

We can technically achieve it, and will support it mid-term, but at the moment re-doing a full download solution just for macOS is not something we want to invest time in, when there's a better download solution on the way for all platforms.

**Comment by garfnodie** [comment]
I almost never use the App Store, didn't for Kiwix either.
